decidim-surveys/db/migrate/20240925124312_add_settings_to_decidim_surveys_surveys.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class AddSettingsToDecidimSurveysSurveys < ActiveRecord::Migration[7.0]
+  class Survey < ApplicationRecord
```

Comment:
For the below to work as intended, you would need this. 
```suggestion
  class Survey < ApplicationRecord
    include Decidim::HasComponent

```

Reply:
Done